### ISSUE TITLE
0.8.x no longer outputs `Compiling <module-name>` when compiling modules

### ISSUE THREAD

**Issue by joneshf** [natefaubion/purescript-psa#48]
It seems like there's no more `Compiling <module>` when using the 0.8.x series of `psa`:

```Console
$ echo 'module Foo where' > Foo.purs
$ rm -fr output/
$ npx --package=purescript-psa@0.7.3 psa Foo.purs
npx: installed 1 in 0.984s
Compiling Foo
           Src   Lib   All
Warnings   0     0     0  
Errors     0     0     0  
$ ls -ahl output/Foo/
total 16
drwxr-xr-x  4 joneshf  staff   128B Jan 23 12:51 ./
drwxr-xr-x  4 joneshf  staff   128B Jan 23 12:51 ../
-rw-r--r--  1 joneshf  staff   128B Jan 23 12:51 externs.cbor
-rw-r--r--  1 joneshf  staff    71B Jan 23 12:51 index.js
$ rm -fr output/
$ npx --package=purescript-psa@0.8.0 psa Foo.purs
npx: installed 1 in 1.187s
           Src   Lib   All
Warnings   0     0     0  
Errors     0     0     0  
$ ls -ahl output/Foo/
total 16
drwxr-xr-x  4 joneshf  staff   128B Jan 23 12:52 ./
drwxr-xr-x  4 joneshf  staff   128B Jan 23 12:52 ../
-rw-r--r--  1 joneshf  staff   128B Jan 23 12:52 externs.cbor
-rw-r--r--  1 joneshf  staff    71B Jan 23 12:52 index.js
```

Notice that the run with 0.7.3 has the line `Compiling Foo` in the output. But the run with 0.8.0 does not have that line. It's the same with 0.8.1. The module is still being compiled. There's no change to which `purs` binary is used, so it shouldn't be an issue like finding a different `purs` between runs.

Is this change intentional? If so, is there a flag to pass that will give us back the `Compiling <module-name>` outputs?

**Comment by natefaubion** [comment]
I suspect that because 0.14 swaps stdin/stderr it will probably show `Compiling` messages for 0.14, but not 0.13? Not totally sure. cc @hdgarrood

**Comment by hdgarrood** [comment]
This is an unintended consequence of the implementation used in #44; previously psa was capturing the stderr output of the inner `purs` process and not doing anything with stdout, so that the "Compiling <module>" messages (which went to stdout) were just printed to the terminal. Now, we're capturing both and just reading the one that contains the information we want (depending on whether we're on >= 0.14.0 or not), and discarding the one we don't. I can send a PR to fix this.

**Comment by natefaubion** [comment]
This was fixed with v0.8.2.

**Comment by joneshf** [comment]
Thanks for addressing this!
